pkg/kv/kvclient/kvcoord/dist_sender.go

```diff
@@ -3225,6 +3226,41 @@ func (ds *DistSender) computeNetworkCost(
 	return cost
 }
 
+// computeWriteReplicationNetworkPaths computes all the network paths taken by
+// the leaseholder when it replicates writes to the other replicas for the given
+// range.
+func (ds *DistSender) computeWriteReplicationNetworkPaths(
```

Comment:
I'd had in mind that we'd directly increment the write replication metrics here, rather than adding it to a `RequestInfo`. After all, after Andrew's changes, all the other network traffic will be directly recorded in metrics rather than being funneled through the controller. Why should this network metric be special?